client/README.md

```diff
@@ -1,5 +1,7 @@
 # InfluxDB Client
 
+[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/influxdb/influxdb/client)
```

Comment:
Yes, let's go with an official one, thanks.

On Wednesday, July 8, 2015, Matthias Lüdtke <notifications@github.com>
wrote:

> In client/README.md
> <https://github.com/influxdb/influxdb/pull/3260#discussion_r34122456>:
>
> > @@ -1,5 +1,7 @@
> >  # InfluxDB Client
> >
> > +[![GoDoc](http://img.shields.io/badge/go-documentation-blue.svg?style=flat-square)](http://godoc.org/github.com/influxdb/influxdb/client)
>
> ...actually only because the official one looks awful on high-res displays.
>
> But now that you mention it, there is also a SVG version of the official
> badge now
> https://godoc.org/github.com/influxdb/influxdb?status.svg
>
> Prefer this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/3260/files#r34122456>.
>